app/main/views/services.py

```diff
@@ -246,6 +245,77 @@ def update_section_submission(service_id, section):
         )
 
 
+@main.route('/submission/g-cloud-7/create', methods=['GET'])
+@login_required
+@flask_featureflags.is_active_feature('CREATE_SERVICE_PAGE')
+def start_new_draft_service():
+    """
+    Page to kick off creation of a new (G7) service.
+    """
+    template_data = main.config['BASE_TEMPLATE_DATA']
+
+    lots = new_service_content.get_question('lot')
+    lots['type'] = 'radio'
```

Comment:
All of the parameters in the SSP are passed directly into the [form-building elements](http://alphagov.github.io/digitalmarketplace-frontend-toolkit/forms/selection-buttons.html), which *almost* works.

The SSP stuff says `type: radios`, which, when passed straight through to the form-building code, renders a broken `<input type="radios">` element.  

I'm not really sure what else depends on using the word `radios`, but it might be nothing.